modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/util/StatementChecker.java

```diff
@@ -371,14 +387,16 @@ private IgniteSchema createSchema() {
         return schema;
     }
 
-    private DynamicTest shouldPass(String name, Throwable exception, Consumer<IgniteRel> check) {
+    private DynamicTest shouldPass(String name, Throwable exception, Consumer<IgniteRel> check, boolean relCheck) {
         return DynamicTest.dynamicTest(name, () -> {
             IgniteSchema schema = initSchema(exception);
             IgniteRel root;
 
             try {
                 root = (IgniteRel) sqlPrepare.prepare(schema, sqlStatement, dynamicParams);
-                checkRel(root, schema);
+                if (relCheck) {
```

Comment:
Why do we need this?

Reply:
yep, i fill appropriate issue https://issues.apache.org/jira/browse/IGNITE-20170

Reply:
Ok i can change it for extended ver, but it need to be implemented in 2-3 classes, i fill the issue and fill apropriate comment to remove it after fix, is it ok ?

Reply:
Sure.